v2/pubsub-to-redis/src/main/resources/pubsub-to-redis-command-spec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mainClass": "com.google.cloud.teleport.v2.templates.PubSubToRedis",
+  "classPath": "/template/pubsub-to-redis/*:/template/pubsub-to-redis/libs/*:/template/pubsub-to-redis/classes",
```

Comment:
it is highly recommended to keep `/template/pubsub-to-redis/pubsub-to-redis-conscrypt.jar` in the first place. 

Our build takes care of putting it there

Reply:
Added /template/pubsub-to-redis/pubsub-to-redis-conscrypt.jar in the beginning.